### ISSUE TITLE
pandas

### ISSUE THREAD

**Issue by jenniferagraham** [ueapy/pythoncourse2023-materials#8]
Do we want a simpler option for showing the import of dates within a csv file? 

(e.g., conversion to datetime with just pd.to_datetime rather than use of a parsing function)

Also, couple of potential errors occur while running the saved notebook? (Check most recent version is pushed to repository)

e.g., pd.datetime -> datetime.datetime (error as pd.datetime not found in OLR/SOI example)

**Comment by Pranjal10052000** [comment]
Yes, making a simpler option instead of parsing function will be handy in pandas. I am not sure about the errors while running the saved notebook. I have not came across any similar error. Hope this helps.
